src/authentication/DemoSiteRequest.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
+ */
+import { hashPassword } from 'sussol-utilities';
+export class DemoSiteRequest {
+  async createActivationURL(username, email, password, repeatPassword) {
+    // Client side validation
+    if (email.length === 0) throw new Error('Enter the E-mail');
```

Comment:
Is it possible to get these error strings from the translation file?

Reply:
Please check https://github.com/sussol/mobile/pull/724/commits/e139e4f1b1b4f6df4df5bb1bb77634d809ec214e